fastplotlib/graphics/features/_base.py

```diff
@@ -254,6 +259,9 @@ def cleanup_array_slice(key: np.ndarray, upper_bound) -> np.ndarray:
     if key.dtype == bool:
         key = np.nonzero(key)[0]
 
+    if key.size < 1:
```

Comment:
I forgot what this fixed :smile:

Reply:
@clewis7 can you jog my memory? Is this when you do something like `line_graphic.colors[xs > 0] = "w"`, but `xs > 0` is an empty array?

Reply:
yes, bc we kept getting an error when trying to set the colors and a behavior had not a occurred so the array was all zeroes, would never give True